File: py/1329.py
while True:
    numeroDeVezes = int(input())
    if numeroDeVezes == 0:
        break
    partidas = input().split(' ')
    maria_wins = 0
    joao_wins = 0
    for l in range(numeroDeVezes):
        if int(partidas[l]) == 0:
            maria_wins += 1
        else:
            joao_wins += 1
    print("Mary won "+str(maria_wins)+" times and John won "+str(joao_wins)+" times")

File: js/1329.js
const argv = process.argv.slice(2);

const N = argv[0];

var maria_wins = 0;
var joao_wins  = 0;

if(N < 1 || N > 10000) return;

for(i = 1; i <= N; i++){
    if(argv[i] === 0) maria_wins++;
    else joao_wins++;
}

console.log(`Mary won ${maria_wins} times and John won ${joao_wins} times`);  